source/release-notes.txt

```diff
@@ -10,6 +10,58 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.34.0
+----------------
+
+New Features:
+
+- Add command-line interface and global configuration (:issue:`COMPASS-6069`, 
+  :issue:`COMPASS-6070`, :issue:`COMPASS-6071`, :issue:`COMPASS-6073`, 
+  and :issue:`COMPASS-6074`)
+
+- Flip the new toolbars feature flag, always show new toolbars 
+  (:issue:`COMPASS-5679`)
+
+- Add autocompleter for aggregation, use autocompleter in import pipeline 
+  modal (:issue:`COMPASS-6175`)
+
+- Add ``protectConnectionStrings`` option (:issue:`COMPASS-6066`)
+
+- Implement ``HADRON_ISOLATED`` through preferences (:issue:`COMPASS-6065`)
+
+- Show icons in the sidebar menus (:issue:`COMPASS-6081`)
+
+- Rebranding components (:issue:`COMPASS-6100`, :issue:`COMPASS-6101`,
+  :issue:`COMPASS-6121`, :issue:`COMPASS-6048`, and :issue:`COMPASS-6187`)
+
+- Layout improvements (:issue:`COMPASS-6148`, :issue:`COMPASS-6150`, and
+  :issue:`COMPASS-5582`)
+
+- Add theme as regular setting (:issue:`COMPASS-6067` and :issue:`COMPASS-5284`)
+
+Bug Fixes:
+
+- Fix map rendering and add e2e tests (:issue:`COMPASS-6131`)
+
+- Reconnect CSFLE client after :dbcommand:`collMod` (:issue:`COMPASS-5989`)
+
+- Improve selection area for insert document editor
+
+- Tell yargs about types for kebab case options
+
+- Add map for collection stats for tab namespace isolation 
+  (:issue:`COMPASS-6146`)
+
+- Open info links in browser (:issue:`COMPASS-6193`)
+
+- Menu not fully showing for field actions, remove old backgrounds 
+  (:issue:`COMPASS-6186`)
+
+- Remove margins from ace brackets
```

Comment:
Similar thought here. I'm not sure what ace brackets here, so maybe we can remove or tweak the language here so it highlights where the user might see this change?

Reply:
Fixed